pliers/extractors/text.py

```diff
@@ -453,98 +446,282 @@ def __init__(self,
         if framework not in ['pt', 'tf']:
             raise(ValueError('''Invalid framework;
                 must be one of 'pt' (pytorch) or 'tf' (tensorflow)'''))
-
-        if encoding_level not in ['token', 'sequence']:
-            raise(ValueError('''Invalid encoding_level;
-                must be one of 'token' or 'sequence'.'''))
-
-        if model_kwargs is None:
-            model_kwargs = {}
-
-        if tokenizer_kwargs is None:
-            tokenizer_kwargs = {}
-
-        self.pretrained_model = pretrained_model_or_path
+        self.pretrained_model = pretrained_model
         self.tokenizer_type = tokenizer
+        self.model_class = model_class
         self.framework = framework
-        self.encoding_level = encoding_level
-        self.pooling = pooling
-        self.model_kwargs = model_kwargs
-        self.tokenizer_kwargs = tokenizer_kwargs
-        model_name = 'BertModel' if self.framework == 'pt' else 'TFBertModel'
+        self.return_metadata = return_metadata
+        self.model_kwargs = model_kwargs if model_kwargs else {}
+        self.tokenizer_kwargs = tokenizer_kwargs if tokenizer_kwargs else {}
 
+        model = model_class if self.framework == 'pt' else 'TF' + model_class
+        self.model = getattr(transformers, model).from_pretrained(
+            pretrained_model, **self.model_kwargs)
         self.tokenizer = transformers.BertTokenizer.from_pretrained(
-            tokenizer, **tokenizer_kwargs)
-        self.model = getattr(transformers, model_name).from_pretrained(
-            pretrained_model_or_path, **model_kwargs)
-
-        super(PretrainedBertEncodingExtractor, self).__init__()
+            tokenizer, **self.tokenizer_kwargs)
+        super(BertExtractor, self).__init__()
+
+    def _mask(self, wds, mask):
+        return wds
+
+    def _preprocess(self, stims, mask):
```

Comment:
Ditto: a very short docstring or comment explaining when this is invoked and what it's doing would be helpful (and the same for `_postprocess`).